### ISSUE TITLE
Multicheckbox and Radios broken in Angular 9

### ISSUE THREAD

**Issue by PartyArk** [ngx-formly/ngx-formly#2071]
I believe a change the way to `renderer2 `is working is causing `Multicheckbox `and other array-like selectors to fail (I'm using Bootstrap, not checked on others).

The issue comes in `FormlyAttributes` selector which is using ngOnChanges to observe the field change. This line is the key:

`this.renderer.setAttribute(this.elementRef.nativeElement, 'id', this.field.id)`

So we're resetting the id of the field. But with array-like form types, they have to be unique, so they have _0, _1, _2 etc. appended in the DOM. That's **not** reflected in the field.id which is just unique on a per `formly-field` basis, so the field.id for all the checkboxes / radios is the same.

I suspect changing the DOM input id to be the same is not what is intended, but it does work in Angular 8 - and I'm not sure why! You can try this for a multicheckbox  -

```
      console.log(this.elementRef.nativeElement);
      console.log("changing id to", this.field.id);
      this.renderer.setAttribute(this.elementRef.nativeElement, 'id', this.field.id);
      console.log(this.elementRef.nativeElement)
```

... and observe no change at all to the element! If I had to guess the change cycle is immediately resetting the id to its "correct" value via the template. Or maybe there's some funny shadow-dom thing going on, or maybe the renderer uses ids to match elements and that's why it's not being picked up. Basically, resetting the ID doesn't work, so it's all ok.

Anyway, now doesn't work in Angular 9. The id on the input element is being updated as expected, so all the ids are the same (wrongly), which means the labels don't match their ids and it's generally now broken.

Thanks!

**Comment by PartyArk** [comment]
Thanks for the speedy response and the excellent library.

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.5.11](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.11) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
